test/index.ts

```diff
@@ -305,6 +305,32 @@ describe('.getToken()', () => {
     });
   });
 
+  it('should not use cached token if forceRefresh=true (promise)', async () => {
+    const gtoken = new GoogleToken(TESTDATA);
+    gtoken.rawToken = {
+      access_token: 'mytoken',
+    };
+    gtoken.expiresAt = new Date().getTime() + 10000;
+    const fakeToken = 'abc123';
+    const scope = createGetTokenMock(200, {access_token: fakeToken});
+    const token = await gtoken.getToken(true);
+    assert.strictEqual(token.access_token, 'abc123');
```

Comment:
In this case, and the next, could you re-use the `fakeToken` variable itself? Maybe another name, like "newToken" would help make it more clear, as well.